db/migrate/20190708153331_migrate_group_list_site_settings.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class MigrateGroupListSiteSettings < ActiveRecord::Migration[5.2]
+  def up
+    migrate_value(:name, :id)
+  end
+
+  def down
+    migrate_value(:id, :name)
+  end
+
+  def migrate_value(from, to)
+    ActiveRecord::Base.transaction do
+      SiteSetting.where(data_type: SiteSettings::TypeSupervisor.types[:group_list]).each do |setting|
```

Comment:
You should only use SQL in migrations for queries. We learned this the hard way when in the future the objects change and the migration no longer runs.

Reply:
Done 👌